exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -452,6 +452,38 @@ class "CesiumRasterOverlayPrim" (
         displayName = "Overlay Render Method"
         doc = "The Cesium default material will give the raster overlay a different rendering treatment based on this selection."
     )
+
+    uniform float cesium:maximumScreenSpaceError = 2.0 (
```

Comment:
Could you update `CHANGES.md` for this PR and https://github.com/CesiumGS/cesium-omniverse/pull/670?

* Added support for WMS raster overlays
* Added raster overlay options: `maximumScreenSpaceError `, `maximumTextureSize`, `maximumSimultaneousTileLoads`, `subTileCacheBytes`